docs/source/api.rst

```diff
@@ -41,3 +41,21 @@ There's also a HAWC client available:
         ids=hero_ids
```

Comment:
sorry, this comment doesn't belong on this file, but can you delete the `client/r/rhawc_0.1.0.tar.gz` file